lib/modules/datasource/conan/index.ts

```diff
@@ -50,6 +55,38 @@ export class ConanDatasource extends Datasource {
     };
   }
 
+  @cache({
+    namespace: `datasource-${datasource}-revisions`,
+    key: ({ registryUrl, packageName }: GetReleasesConfig) =>
+      // TODO: types (#7154)
+      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+      `${registryUrl}:${packageName}`,
+  })
+  override async getDigest(
+    { registryUrl, packageName, currentDigest }: DigestConfig,
+    newValue?: string
+  ): Promise<string | null> {
+    if (is.undefined(currentDigest) || is.undefined(registryUrl)) {
+      return null;
+    } else {
+      const url = ensureTrailingSlash(registryUrl);
+      const depName = packageName.split('/')[0];
+      const userAndChannel = '/' + packageName.split('@')[1];
+      const packageRoute = newValue
+        ? `${depName}/${newValue}${userAndChannel}`
+        : packageName.replace('@', '/');
```

Comment:
this isn't my suggestion, please follow what i suggest.

Reply:
Sorry can you link it, I cannot see which one you are talking about